evol/population.py

```diff
@@ -65,10 +65,10 @@ def __copy__(self):
         result.documented_best = self.documented_best
         return result
 
-    def __iter__(self):
+    def __iter__(self) -> [Individual]:
```

Comment:
One last thing: shouldn't this be something like `Generator[Individual, None, None]`?

Reply:
Yes, it does. `self.individuals` is `List[int]`, but it doesn't return `self.individuals`: `return self.individuals.__iter__()` which is an `Iterator[Individual]`.

Reply:
d0h. making the change now.